doc/user-guides/acrn-dm-parameters.rst

```diff
@@ -132,6 +156,9 @@ Here are descriptions for each of these ``acrn-dm`` command line parameters:
 
        Example: ``-p  "1:2"`` means pin the 2nd physical cpu to 1st vcpu in UOS
 
+   * - :kbd:`--part_info <part_info_name>`
```

Comment:
Documentation for ``--vtpm2`` argument is missing.
Also for ``--vmcfg`` and ``--dump``.